web/static/js/components/idea_controls.jsx

```diff
@@ -21,6 +22,11 @@ const IdeaControls = props => {
   const highlightTitle = isHighlighted ? "De-Highlight Idea for Participants" : "Announce Idea to Channel"
 
   function renderIcons() {
+    if (stage !== "idea-generation" && category !== "action-item") {
```

Comment:
per the comment above, the condition here should just be whether it's the voting stage